### PR TITLE
Add repository snapshots to export all [SCI-4738]

### PR DESCRIPTION
**Pull request by okriuchykhin**
Jira ticket: [SCI-4738](https://biosistemika.atlassian.net/browse/SCI-4738)

### What was done
Add repository snapshots to export all

### CODE REVIEW

File: app/models/report.rb [hunk 1]
```diff
@@ -131,17 +131,13 @@ def self.generate_whole_project_report(project, current_user, current_team)
...
-          gen_element_content(my_module,
-                              my_module.repository_rows.select(:repository_id)
-                                       .distinct.map(&:repository),
-                              'my_module_repository', true, 'asc')
-
-        modules +=
-          gen_element_content(my_module, nil, 'my_module', true, nil,
-                              module_children)
+        module_children += gen_element_content(my_module, nil, 'my_module_activity', true, 'asc')
+
```
Comment (radiokills) [inline app/models/report.rb:135]: what's this newline madness :D ?

File: app/models/repository.rb [hunk 2]
```diff
@@ -83,6 +83,12 @@ def self.search(
     end
   end
 
+  def self.assigned_in_project(project)
```
Comment (radiokills) [inline app/models/repository.rb:86]: shouldn't this be scope?

Comment (radiokills) [reply app/models/repository.rb:86]: and maybe more proper naming 'assigned_to_project

File: app/models/repository_snapshot.rb [hunk 3]
```diff
@@ -15,6 +15,15 @@ class RepositorySnapshot < RepositoryBase
   validates :status, presence: true
   validate :only_one_selected_for_my_module, if: ->(obj) { obj.changed.include? :selected }
 
+  def self.assigned_in_project(project)
```
Comment (radiokills) [inline app/models/repository_snapshot.rb:18]: same here

File: app/models/repository.rb (hunk 2 above)
Comment (okriuchykhin) [reply app/models/repository.rb:86]: they are not directly assigned to a project

**Pull request closed without merging**